### PR TITLE
Add second set of PD Application Banners

### PR DESCRIPTION
**Pull request by molly-moen**
# Description
Add P2 set of professional development application banners as specified in [Jira](https://codedotorg.atlassian.net/browse/PLC-656). The banners use the same banner element from https://code.org/educate/professional-learning/middle-high, with different text and a slightly smaller font (to fix the text). Also updated previous home banner to say "...Applications **are** now open..." instead of "...Applications now open..."

New banners are on the following pages:
 https://code.org/educate/curriculum/middle-school
Screen Shot 2020-01-13 at 4 19 15 PM

 https://code.org/educate/curriculum/high-school
Screen Shot 2020-01-13 at 4 18 56 PM

https://code.org/hourofcode/overview 
Screen Shot 2020-01-13 at 4 21 56 PM

## Links

- [jira](https://codedotorg.atlassian.net/browse/PLC-656)
- [P1 banners PR](https://github.com/code-dot-org/code-dot-org/pull/32629)

## Testing story
Validated banners look as spec'd in Jira on multiple browsers, and that banners still look reasonable when window is resized.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: pegasus/sites.v3/code.org/views/professional_learning_apply_banner.haml [hunk 1]
```diff
@@ -16,8 +19,10 @@
         .arrowcontainer.desktop-feature{style: "float:left; width: 7%"}
           .arrow{style: "width: 0; height: 0; border-style: solid; border-width: 50px 0 50px 25px; border-color: transparent transparent transparent #ebe8f1"}
         .textcontainer{style: "float:left; width: 93%"}
-          .text{style: "color: white;  font-size: 18px; padding: 20px 10px 20px 10px; text-align: center; line-height: 1.5"}
-            - if params[:nominated]
+          .text{style: "color: white;  font-size: #{font_size}; padding: #{padding}; text-align: center; line-height: 1.5"}
+            - if use_sign_up_text
+              Sign up for Professional Learning! Applications now open for Middle and High School teachers.
```
Comment (islemaster) [inline pegasus/sites.v3/code.org/views/professional_learning_apply_banner.haml]: Something I should have caught in the last PR: There's no verb in the second sentence.  @agealy can we change this to "Applications _are_ now open..."?

Comment (agealy) [reply pegasus/sites.v3/code.org/views/professional_learning_apply_banner.haml]: Sure, I'm ambivalent. The current verbless phrasing was what came through the marketing and programs approved request

Comment (molly-moen) [reply pegasus/sites.v3/code.org/views/professional_learning_apply_banner.haml]: would we want to update the homepage text as well? (it's the same "Applications now open...")

Comment (islemaster) [reply pegasus/sites.v3/code.org/views/professional_learning_apply_banner.haml]: :+1: I vote yes.

Comment (molly-moen) [reply pegasus/sites.v3/code.org/views/professional_learning_apply_banner.haml]: updated text

**Review by islemaster — APPROVED** [review]
:+1: Looks great, thank you!

**Review by agealy — APPROVED** [review]

**Pull request merged**